### DISCUSSION TITLE
Query Current Connected User?

### DISCUSSION THREAD

**Discussion by jdhoang** [apache/airflow#28971] [Q&A]
I wrote a plugin to extend the UI by adding a custom web page - this works fine but wanted to check current connected user to determine access.  I've been searching but not able to find how to do this.  Would appreciate any advice!  Thank you!

**Comment by potiuk** [comment]
I believe you need to look in Flask or FlaskAppBuilder.  By looking at the source code, the user should be available in Flask's global g variable as `g.user` - but I do not know Flask at all - I just looked at the source code of airlrow. So you should likely experiment an debug and verify it

**Reply by jdhoang** [reply]
Thank you for your help, Jarek!
